### PR TITLE
update - companies, fullstack

### PR DESCRIPTION
**Pull request by rbalet**
I change unDraw to undraw, I know, the name is not right, but since when you download images from undraw the name of the file are named like this `undraw_...` it will avoid having an error when using those image as assets.

### CODE REVIEW

File: packages/companies/companies.txt [hunk 1]
```diff
@@ -1055,3 +1055,4 @@ Zhejiang Materials Industry Group
 Ziggo
 Zoho
 Zurich Insurance Group
+Adblock
```
Comment (Jason3S) [inline packages/companies/companies.txt]: This line is duplicate.

File: packages/fullstack/fullstack.txt [hunk 2]
```diff
@@ -1,10 +1,13 @@
 actioned
+allday
 androiddelay
 antialiased
 autofetch
 autofill
 autogenerated
 autoprefixer
+autosize
+autosize
```
Comment (Jason3S) [inline packages/fullstack/fullstack.txt]: This one is also duplicate.

File: packages/fullstack/fullstack.txt [hunk 3]
```diff
@@ -15,6 +18,7 @@ ckeditor
 classlist
 codelyzer
 collapsable
+combinatorics
```
Comment (Jason3S) [inline packages/fullstack/fullstack.txt]: This term is ideally in [softwareTerms.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/master/packages/software-terms/softwareTerms.txt)

**Comment by Jason3S** [comment]
Thank you for helping out!

**Comment by rbalet** [comment]
@Jason3S Ok, done

**Comment by Jason3S** [comment]
Thank you!

**Pull request merged**